kafka-ui-react-app/src/components/Connect/Details/Actions/__tests__/Actions.spec.tsx

```diff
@@ -35,8 +35,12 @@ const expectActionButtonsExists = () => {
 };
 const afterClickDropDownButton = async () => {
```

Comment:
are we using this function ? cause it seems to be deleted

Reply:
I added a new function because there pops up a warning that 'button cannot be located inside another button'

Reply:
in the dropdown, I just added a div with onClick and now it works correctly and there is no warning in the tests

Reply:
and yes we are using this function because the old drop down still works for deletion